src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/NullableAttributes.cs

```diff
@@ -3,6 +3,8 @@
 
 namespace System.Diagnostics.CodeAnalysis
 {
+// These attributes already shipped with .NET Core 3.1 in System.Runtime
+#if !NETCOREAPP3_0 && !NETCOREAPP3_1 && !NETSTANDARD2_1
```

Comment:
I see.  In regards to my earlier comment / question, you're now including this file for all < 5.0 builds, but in 3.x / .NET Standard 2.1 builds you're excluding the attributes that already shipped.